src/isar/apis/models/models.py

```diff
@@ -11,7 +11,7 @@ class StepResponse(BaseModel):
 
 class TaskResponse(BaseModel):
     id: str
-    tag_id: Optional[str]
+    tag_id: Optional[str] = None
```

Comment:
Why has this been added and have you properly tested this one?

The reason I ask is that for dataclasses atleast it is not supported to have a default value between fields that do not have one. It is probably different for pydantic but just checking to know you have thought about it.